static/app/components/modals/demoSignUp.tsx

```diff
@@ -0,0 +1,83 @@
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import {ModalRenderProps} from 'app/actionCreators/modal';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import HighlightModalContainer from 'app/components/highlightModalContainer';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import getCookie from 'app/utils/getCookie';
+
+type Props = ModalRenderProps;
+
+const ForcedTrialModal = ({closeModal}: Props) => {
+  const email = localStorage.getItem('email');
+  const queryParameter = email ? `?email=${email}` : '';
+  const extraQueryString = getCookie('extra_query_string');
```

Comment:
@roggenkemper a lot of this logic is copied from `DemoHeader`, could you make some utility functions for this that we can share between components? We probably could create some demo/sandbox utility function.